### PR TITLE
Hotfix Release - 6 May

### PR DESCRIPTION
**Pull request by aman-tunerlabs**

### CODE REVIEW

File: module/organisations/helper.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ * author : Aman
+ * Date : 05-May-2020
+ * Description : Organisation details
+ */
+
+const sunbirdService = 
+require(ROOT_PATH +"/generics/services/sunbird");
+const sessionHelpers = require(ROOT_PATH+"/generics/helpers/sessions");
+
+module.exports = class UserProfileHelper {
```
Comment (aks30) [inline module/organisations/helper.js]: @aman-tunerlabs - Why user profile helper ?

File: generics/services/sunbird.js [hunk 2]
```diff
@@ -327,12 +327,52 @@ var indexSync = async function ( syncData,token ) {
...
 
+/**
+  * Lists of organisations
+  * @function
+  * @name organisationList
+  * @param requestData - requested data
+  * @returns {Promise}
+*/
+
+var organisationList = async function (requestData) {
```
Comment (aks30) [inline generics/services/sunbird.js:338]: @aman-tunerlabs - No token required?

File: config/globals.js [hunk 3]
```diff
@@ -181,4 +181,8 @@ module.exports = function () {
       })
     }
   });
+
+  const organisationHelper = require(ROOT_PATH+"/module/organisations/helper");
```
Comment (aks30) [inline config/globals.js]: @aman-tunerlabs - I am thinking, should we set it from the list helper function, so whenever the first time list is called, it checks in session, if yes, return from there, else make API call, set in session the same data and return from there. Let me know

File: generics/services/sunbird.js (hunk 2 above)
Comment (aman-tunerlabs) [reply generics/services/sunbird.js:338]: @aks30  Not required

**Pull request merged**